src/app/pages/send/ordinal-inscription/hooks/use-send-inscription-fees-list.ts

```diff
@@ -0,0 +1,101 @@
+import { useMemo } from 'react';
+
+import { BtcFeeType, btcTxTimeMap } from '@shared/models/fees/bitcoin-fees.model';
+import { createMoney } from '@shared/models/money.model';
+
+import { baseCurrencyAmountInQuote } from '@app/common/money/calculate-money';
+import { formatMoneyPadded, i18nFormatCurrency } from '@app/common/money/format-money';
+import { FeesListItem } from '@app/components/bitcoin-fees-list/bitcoin-fees-list';
+import { useCurrentNativeSegwitUtxos } from '@app/query/bitcoin/address/address.hooks';
+import { useAverageBitcoinFeeRates } from '@app/query/bitcoin/fees/fee-estimates.hooks';
+import { TaprootUtxo } from '@app/query/bitcoin/ordinals/use-taproot-address-utxos.query';
+import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/market-data.hooks';
+import { useCurrentAccountNativeSegwitSigner } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+import { selectInscriptionTransferCoins } from '../coinselect/select-inscription-coins';
+
+interface UseSendInscriptionFeesListArgs {
+  recipient: string;
+  utxo: TaprootUtxo;
+}
+export function useSendInscriptionFeesList({ recipient, utxo }: UseSendInscriptionFeesListArgs) {
+  const createNativeSegwitSigner = useCurrentAccountNativeSegwitSigner();
+  const { data: nativeSegwitUtxos } = useCurrentNativeSegwitUtxos();
+
+  const btcMarketData = useCryptoCurrencyMarketData('BTC');
+  const { avgApiFeeRates: feeRates, isLoading } = useAverageBitcoinFeeRates();
+
+  const feesList: FeesListItem[] = useMemo(() => {
+    function getFiatFeeValue(fee: number) {
+      return `~ ${i18nFormatCurrency(
+        baseCurrencyAmountInQuote(createMoney(Math.ceil(fee), 'BTC'), btcMarketData)
+      )}`;
+    }
+
+    const nativeSegwitSigner = createNativeSegwitSigner?.(0);
+
+    if (!feeRates || !nativeSegwitUtxos || !nativeSegwitSigner) return [];
+
+    const highFeeResult = selectInscriptionTransferCoins({
+      recipient,
+      inscriptionInput: utxo,
+      nativeSegwitUtxos,
+      changeAddress: nativeSegwitSigner?.payment.address!,
+      feeRate: feeRates.fastestFee.toNumber(),
+    });
+
+    const standardFeeResult = selectInscriptionTransferCoins({
+      recipient,
+      inscriptionInput: utxo,
+      nativeSegwitUtxos,
+      changeAddress: nativeSegwitSigner?.payment.address!,
```

Comment:
Shouldn't be necessary because of check on L37 right?

```suggestion
      changeAddress: nativeSegwitSigner.payment.address!,
```